packages/overlay/src/integrations/sentry/components/performance/TransactionDetail.tsx

```diff
@@ -0,0 +1,168 @@
+import { useMemo, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { ReactComponent as Sort } from '~/assets/sort.svg';
+import { ReactComponent as SortDown } from '~/assets/sortDown.svg';
+import classNames from '~/lib/classNames';
+import Breadcrumbs from '~/ui/Breadcrumbs';
+import { TRANSACTION_SUMMARY_SORT_KEYS, TRANSACTION_SUMMARY_TABLE_HEADERS } from '../../constants';
+import { useSentryEvents } from '../../data/useSentryEvents';
+import { useSentryHelpers } from '../../data/useSentryHelpers';
+import { SentryTransactionEvent } from '../../types';
+import { getFormattedDuration } from '../../utils/duration';
+import { truncateId } from '../../utils/misc';
+import DateTime from '../DateTime';
+
+export default function TransactionDetail({ showAll }: { showAll: boolean }) {
+  const { name } = useParams();
+
+  const events = useSentryEvents();
+  const helpers = useSentryHelpers();
+
+  const [sort, setSort] = useState({
+    active: TRANSACTION_SUMMARY_SORT_KEYS.timestamp,
+    asc: false,
+  });
+
+  const toggleSortOrder = (type: string) =>
+    setSort(prev =>
+      prev.active === type
+        ? {
+            active: type,
+            asc: !prev.asc,
+          }
+        : {
+            active: type,
+            asc: false,
+          },
+    );
+
+  type TransactionsInfoComparator = (a: SentryTransactionEvent, b: SentryTransactionEvent) => number;
+  type TransactionsSortTypes = (typeof TRANSACTION_SUMMARY_SORT_KEYS)[keyof typeof TRANSACTION_SUMMARY_SORT_KEYS];
```

Comment:
Move to `constants.ts`?

Reply:
Lets keep it here for now. Will move all these in types file later for all pages.